projects/demo/src/modules/components/editor/font/examples/3/font-size-tool/font-size-tool.styles.less

```diff
@@ -0,0 +1,21 @@
+@import 'taiga-ui-local';
+
+.preview {
+    border: 1px solid var(--tui-base-03);
+}
+
+::ng-deep [data-appearance='font-size'] {
```

Comment:
Could we use the following code?
```html
<button
  tuiIconButton
  type="button"
  appearance="icon"
  icon="tuiIconMinus"
  size="s"
></button>

<button
  tuiIconButton
  type="button"
  appearance="icon"
  icon="tuiIconPlus"
  size="s"
></button>
```

It better to keep example as simple as possible: user should focus on main concepts.

Reply:
fixed